docs/migration/manual-migration.mdx

```diff
@@ -38,6 +40,24 @@ grep "module.exports" src/ -r
 
 :::
 
+#### Updating the project's `package.json`
+
+This transition will also require your project to be declared as a `module`. In
+order to do so, you can add the following to your `package.json`:
+
+```json
+  // ...
+  "name": "my-project",
+  // highlight-next-line
+  "type": "module",
+  //...
+```
```

Comment:
**issue:** I don't see the value of such information here.\
When migrating from 2.x, integrators will start with a fresh 3.x skeleton. The skeleton already has it.

**suggestion:** remove this part.

**suggestion:** improve the 3.3 -> 3.4 migration guide's section "[Update package.json commands](https://deploy-preview-877--heuristic-almeida-1a1f35.netlify.app/docs/3.x/upgrade/migration-guides/3.3-3.4#update-packagejson-commands)" to make it more obvious (it looks like it was overlooked in your context).
- update title
- add bullet points: add "type" field, update commands

Reply:
Applied your comments and moved the section to 3.3->3.4 migration guide.